app/views.py

```diff
@@ -0,0 +1,143 @@
+
+import os
+from user import User
+from recipes import RecipeCat
+from flask import Flask, render_template, request, session, redirect, url_for
+
+from app import app
+
+app.secret_key = os.urandom(24)
```

Comment:
This should be an environemt setting and should not be pushed to git. Eg. Create a .env file and read the env variable using `os.getenv(<variable_name>, <default>)` or you could use `Decouple`